platform/ui/src/components/Toolbox/Toolbox.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useRef } from 'react';
+import { useToolbar } from '@ohif/core';
+import { ToolboxUI } from './';
+import { useToolbox } from '../../contextProviders';
+
+/**
+ * This component might seem out of place in the UI, but it plays a crucial role in
```

Comment:
Need a sentence saying what it is  'A toolbox is a collection of buttons and commands that they invoke, used to provide custom control panels to users' - that is my take on what this is, but I might be completely off base.
Also, the UI is supposed to have generic UI components, and this one is generic, it is just that it is generic in that it interacts with the services/commands in a generic fashion.   There isn't anything in this that couldn't be used for a completely different type of app, so by that definition it belongs in the ui components.